share/instruments/instruments.xml

```diff
@@ -6639,14 +6639,18 @@
                   <description>Tuned hand bells.</description>
                   <musicXMLid>pitched-percussion.handbells</musicXMLid>
                   <staves>2</staves>
-                  <clef>G</clef>
+                  <transposingClef>G</transposingClef>
+                  <concertClef>G8va</concertClef>
                   <bracket>1</bracket>
                   <bracketSpan>2</bracketSpan>
                   <barlineSpan>2</barlineSpan>
-                  <clef staff="2">F</clef>
+                  <transposingClef staff="2">F</transposingClef>
+                  <concertClef staff="2">F8va</concertClef>
                   <aPitchRange>36-97</aPitchRange>
                   <pPitchRange>36-97</pPitchRange>
                   <singleNoteDynamics>0</singleNoteDynamics>
+                  <transposeDiatonic>7</transposeDiatonic>
+                  <transposeChromatic>12</transposeChromatic>
                   <Channel>
                         <!--MIDI: Bank 0, Prog 112; MS General: Tinker Bell-->
                         <program value="112"/> <!--Tinkle Bell-->
```

Comment:
`Tinker Bell` vs. `Tinkle Bell`?

Reply:
"Tinker Bell" is the name of a [fictional character](https://en.wikipedia.org/wiki/Tinker_Bell) so it's probably a mistake here, although not necessarily since I could imagine this sound being used to represent that character (a fairy).

Anyway, if it's a mistake then it is a mistake in the MS General SoundFont rather than the spreadsheet or instruments.xml, which is correctly reporting the name used in the SoundFont. "Tinkle Bell" on the `<program>` line refers to the name in General MIDI / Roland GS which is [often different](https://docs.google.com/spreadsheets/d/1SwqZb8lq5rfv5regPSA10drWjUAoi65EuMoYtG-4k5s/edit#gid=133112496) to the name used in the SoundFont.

Reply:
Try to google "Tinkle Bell" and be surprised...